conf/application.conf

```diff
@@ -77,3 +77,4 @@ microservice {
   }
 }
 
+auth.confidenceLevel = 200
```

Comment:
my understanding that we only accepting CL of 250, but worth to raise this question